### PR TITLE
virt-v2v: Replace the Get-Disk and Set-Disk with diskpart

### PR DESCRIPTION
**Pull request by mnecas**
Issue: The Get-Disk and Set-Disk do not support dynamic disks. https://learn.microsoft.com/en-us/powershell/module/storage/get-disk?view=windowsserver2022-ps

### CODE REVIEW

**Review by yaacov — APPROVED** [review]
I don't speak ps1, lgtm

File: virt-v2v/cold/scripts/windows/9999-restore_config.ps1 [hunk 1]
```diff
@@ -6,10 +6,28 @@ Write-Output ('') >> $logFile
...
+    # Iterate through each matching disk
+    foreach ($disk in $disks) {
+        $diskNumber = $disk.Index  # Get the disk index which corresponds to the disk number
+
+        # Create a diskpart script to set the disk online
+        $diskpartScript = @"
+select disk $diskNumber
+online disk
+attributes disk clear readonly
+"@
```
Comment (fabiand) [inline virt-v2v/cold/scripts/windows/9999-restore_config.ps1:26]: This feels quie risky to me.

Take an index from one tool/api, and use it in a different tool/api.

This is just waiting for indexes to be out of sync for whatever reason.

**Review by fabiand — CHANGES_REQUESTED** [review]
Mostly requetsing change to avoid that we accidentally merge it.

File: virt-v2v/cold/scripts/windows/9999-restore_config.ps1 (hunk 1 above)
Comment (mnecas) [reply virt-v2v/cold/scripts/windows/9999-restore_config.ps1:26]: 100% agree and I would like to not use the diskpart

**Comment by fabiand** [comment]
Can we close this?

**Pull request closed without merging**